scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -401,7 +401,7 @@ download_cncli() {
   echo -e "\n  Downloading CNCLI..."
   rm -rf /tmp/cncli-bin && mkdir /tmp/cncli-bin
   pushd /tmp/cncli-bin >/dev/null || err_exit
-  cncli_asset_url="$(curl -s https://api.github.com/repos/${G_ACCOUNT}/cncli/releases/latest | jq -r '.assets[].browser_download_url' | grep 'ubuntu22.*.linux-musl.tar.gz')"
+  cncli_asset_url="$(curl -s https://api.github.com/repos/cardano-community/cncli/releases/latest | jq -r '.assets[].browser_download_url' | grep 'ubuntu22.*.linux-musl.tar.gz')"
```

Comment:
@Fuma419 
I went through another series of testing for forked builds. Even if the CNCLI repository is forked by the SPO there are no releases, so no assets[] or browser_download_url exists in the forked repository. 

This looks to be the most sane choice, setting it to the upstream repository.

Reply:
I will keep this change in this branch. Thanks for all your help @TrevorBenson. I'm learning a lot reading through your analysis.